shoulder/__main__.py

```diff
@@ -21,82 +21,17 @@
 # SOFTWARE.
 
 import os
-import glob
 
-from shoulder.parser.armv8_xml_parser import ArmV8XmlParser
-from shoulder.generator.c_header_generator import CHeaderGenerator
-from shoulder.generator.cxx_header_generator import CxxHeaderGenerator
 from shoulder.config import config
 from shoulder.logger import logger
+from shoulder.parser import *
+from shoulder.generator import *
 from shoulder.filters import *
 
-def printReg(reg):
```

Comment:
I can't remember if I added this or you did, but we should be able to use ```str(reg)``` to print a register via its custom __str__() function

Reply:
I added it when I was experimenting with the parser but forgot to take it out. I agree, the register functions are cleaner.